rpm/libcap-ng.spec

```diff
@@ -117,7 +115,6 @@ rm -rf $RPM_BUILD_ROOT
 %{python3_sitearch}/capng.py*
 
 %files utils
-%doc LICENSE
-%doc COPYING
```

Comment:
According to fedora spec and license field of the package this should "%licence COPYING" because this is GPLv2 not LGPLv2 like the other packages.